### PR TITLE
Documenting first, last, skip and zip methods in collections

### PR DESCRIPTION
**Pull request by lorenzo**

### CODE REVIEW

File: en/core-libraries/collections.rst [hunk 1]
```diff
@@ -657,6 +708,32 @@ the first argument, starting from the position passed in the second argument::
 
 Positions are zero-based, therefore the first position number is ``0``.
 
+.. php:method:: skip(int $positions)
+
+While the second argument of ``take()`` can help you skip some elements before
+getting them form the collection, you can also use ``skip()`` for the same
```
Comment (bcrowe) [inline en/core-libraries/collections.rst]: **from** the collection

File: en/core-libraries/collections.rst [hunk 2]
```diff
@@ -657,6 +708,32 @@ the first argument, starting from the position passed in the second argument::
...
+purpose as a way to take the rest of the elements after certain position::
```
Comment (bcrowe) [inline en/core-libraries/collections.rst]: after **a** certain

File: en/core-libraries/collections.rst [hunk 3]
```diff
@@ -657,6 +708,32 @@ the first argument, starting from the position passed in the second argument::
...
+
+    $collection = new Collection([1, 2, 3, 4]);
+    $allExceptFirstTwo = $collection->skip(2)->toList(); // [3, 4]
+
+.. php:method:: first()
+
+One of the most common usages of ``take()`` is getting the first element in the
```
Comment (bcrowe) [inline en/core-libraries/collections.rst]: s/usages/uses

**Comment by lorenzo** [comment]
Thanks @bcrowe

**Pull request merged**